apps/src/code-studio/components/AssetUploader.jsx

```diff
@@ -24,14 +27,19 @@ var AssetUploader = React.createClass({
   },
 
   render: function () {
+    let api = this.props.useFilesApi ? filesApi : assetsApi;
+    let url = api.getUploadUrl();
+    let uploadDone = api.makeUploadDoneCallback(this.props.onUploadDone);
+    let uploadStart = api.makeUploadStartCallback(this.props.onUploadStart);
```

Comment:
I wonder if these should be called `wrapUpload<Event>Callback`.  Also, is there any way to check that we're not forgetting these elsewhere?